includes/multi-currency/Settings.php

```diff
@@ -57,410 +47,33 @@ public function __construct( MultiCurrency $multi_currency ) {
-		$available_currencies = $this->multi_currency->get_available_currencies();
-		$currency             = $available_currencies[ strtoupper( $current_section ) ];
-		$default_currency     = $this->multi_currency->get_default_currency();
-
-		?>
-		<tr valign="top">
-			<th scope="row" class="titledesc">
-				<label for="wcpay_multi_currency_preview_default"><?php echo esc_html( $default_currency->get_name() ); ?></label>
-			</th>
-			<td class="forminp forminp-text">
-				<span style="line-height:30px"><?php echo esc_html( $default_currency->get_symbol() ); ?></span>
-				<input name="wcpay_multi_currency_preview_default" id="wcpay_multi_currency_preview_default" type="text" value="20.00" placeholder="20.00">
-			</td>
-		</tr>
-		<tr valign="top">
-			<th scope="row" class="titledesc">
-				<label for="wcpay_multi_currency_preview_converted"><?php echo esc_html( $currency->get_name() ); ?></label>
-			</th>
-			<td>
-				<div id="wcpay_multi_currency_preview_converted">
-					<span style="display:inline-block;"></span>
-				</div>
-				<input type="hidden"
-					name="<?php echo esc_attr( $this->id . '_is_zero_decimal' ); ?>"
-					value="<?php echo esc_attr( $available_currencies[ $currency->get_code() ]->get_is_zero_decimal() ); ?>"
-				/>
-				<input type="hidden"
-					name="<?php echo esc_attr( $this->id . '_automatic_exchange_rate' ); ?>"
-					value="<?php echo esc_attr( $available_currencies[ $currency->get_code() ]->get_rate() ); ?>"
-				/>
-			</td>
-		</tr>
-		<?php
-	}
-
-	/**
-	 * Returns the settings for the single currency.
-	 *
-	 * @param Currency $currency The currency object we're getting settings for.
-	 *
-	 * @return array Array of settings.
-	 */
-	public function get_currency_setting( $currency ) {
-		$available_currencies = $this->multi_currency->get_available_currencies();
-		$default_currency     = $this->multi_currency->get_default_currency();
-		$page_id              = $this->id . '_single_currency';
-
-		if ( $default_currency->get_is_zero_decimal() ) {
-			$default_rate   = '1,000' . $default_currency->get_code();
-			$converted_rate = number_format( $available_currencies[ $currency->get_code() ]->get_rate() * 1000, 4 ) . ' ' . $currency->get_code();
-		} else {
-			$default_rate   = '1' . $default_currency->get_code();
-			$converted_rate = number_format( $available_currencies[ $currency->get_code() ]->get_rate(), 4 ) . ' ' . $currency->get_code();
-		}
-
-		$last_updated = ! is_null( $available_currencies[ $currency->get_code() ]->get_last_updated() )
-			? $this->format_last_updated_date( $available_currencies[ $currency->get_code() ]->get_last_updated() )
-			: 'Error - Unable to fetch automatic rate for this currency';
-
-		$exchange_rate_options = [
-			'automatic' => sprintf(
-				/* translators: %1$s: default currency rate, %2$s: new currency exchange rate, %3$s: time rates were last updated. */
-				__( 'Fetch rate automatically. Current rate: %1$s = %2$s (Last updated: %3$s)', 'woocommerce-payments' ),
```

Comment:
It looks like this text is no longer used and the text from the original design is used instead, looks like it was probably missed in the previous PR. This was changed due to the rates are not updated hourly. Also the `format_last_updated_date` method was used to display the date in the site's date & time format so that the merchant wouldn't have to do any time zone conversions and would be able to tell if something was broken if it had been days since an update.

Reply:
Yes, I've taken the Figma's as the reference for the strings in the previous (Single settings page) PR, and there was the date shown as the last update indicator, The datetime comes in as a timestamp (UTC) and I used `moment` library to show it in user's timezone in that PR. So if the rate won't update in days, still the user will see that and investigate it.

Reply:
Addressed in 16ca714.